### PR TITLE
Upgrade Framer Motion

### PR DESCRIPTION
**Pull request by amcdnl**
This PR:

- Upgrades framer-motion
- Refactors useInterpolate to work with latest framer-motion

### CODE REVIEW

File: src/RadialBarChart/RadialBarSeries/MotionBar.tsx [hunk 1]
```diff
@@ -8,26 +8,26 @@ export const MotionBar = ({ custom, transition, arc, ...rest }) => {
...
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
   useEffect(() => {
     const interpolator = interpolate(currentYRef.current, custom.enter.y);
+    const prevSpring = spring.get();
 
-    spring.set(1);
+    if (transition?.delay) {
+      setTimeout(() => {
```
Comment (amcdnl) [inline src/RadialBarChart/RadialBarSeries/MotionBar.tsx:19]: Do we want to clear the timeout in the event it gets multiple queued?

Comment (ghsteff) [reply src/RadialBarChart/RadialBarSeries/MotionBar.tsx:19]: oh yea good call

File: src/PieChart/PieArcSeries/useInterpolate.ts [hunk 2]
```diff
@@ -17,42 +15,34 @@ export const useInterpolate = ({ data, animated, arc }) => {
...
-    from: 0,
-    to: 1
-  });
+  const prevData = useRef(exit);
+  const d = useMotionValue(exit);
+  const spring = useSpring(0, DEFAULT_TRANSITION);
+
+  // delay the initial animation by 100ms
+  useEffect(() => {
+    setTimeout(() => {
```
Comment (amcdnl) [inline src/PieChart/PieArcSeries/useInterpolate.ts:24]: Probably want to clear this one too

**Review by ghsteff — APPROVED** [review]
🚀

**Pull request merged**